etna/stories/blocks.py

```diff
@@ -243,8 +243,8 @@ class ContentSectionBlock(SectionDepthAwareStructBlock):
 
     class Meta:
         label = "Section"
-        template = "insights/blocks/section.html"
+        template = "stories/blocks/section.html"
 
 
-class InsightsPageStreamBlock(blocks.StreamBlock):
+class StoriesPageStreamBlock(blocks.StreamBlock):
```

Comment:
```suggestion
class StoryPageStreamBlock(blocks.StreamBlock):
```

Reply:
This is now changed to reflect categorised name `Article`